composer_deploy.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Validate arguments passed to the script
+if [[ $# -eq 1 && -n "$1" && -d "$1" ]]; then
+    script_path=$(cd $(dirname $0) && pwd)
+    project_path=$1
+else
+    echo "Usage: bash composer_deploy.sh <project_path>"
+    exit 1
+fi
+
+# Copy only required files to the project directory
+mkdir -p "$1"/docker
+cp -r "${script_path}"/elasticsearch "$1"/docker/
```

Comment:
It's possible to envisage symlink ? It would allow to keep the script updated during the update

Reply:
These files are automatically updated as the script is triggered by Composer when an install/update is performed.

```
  "scripts": {
    "post-install-cmd": "bash vendor/emakinafr/docker-magento2/composer_deploy.sh .",
    "post-update-cmd": "bash vendor/emakinafr/docker-magento2/composer_deploy.sh ."
  }
```